.github/workflows/trivy_scan.yml

```diff
@@ -35,6 +39,33 @@ jobs:
         docker pull antrea/antrea-agent-ubuntu:${{ steps.find-antrea-greatest-version.outputs.antrea_version }}
         docker pull antrea/antrea-controller-ubuntu:latest
         docker pull antrea/antrea-controller-ubuntu:${{ steps.find-antrea-greatest-version.outputs.antrea_version }}
+    - name: Install Trivy
+      uses: aquasecurity/setup-trivy@v0.2.1
+    - name: Get current UTC date
+      id: date
+      run: echo "date=$(date -u +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+    - name: Restore Trivy DB cache
+      if: ${{ !inputs.no-cache }}
+      id: restore-db-cache
+      uses: actions/cache/restore@v4
+      with:
+        path: ${{ github.workspace }}/.cache/trivy
+        key: cache-trivy-db-${{ steps.date.outputs.date }}
+        restore-keys: cache-trivy-db-
```

Comment:
What's the purpose of this prefix-match if it downloads the DB once when there is no exact-match?

Reply:
My understanding (which may be wrong) is that `trivy image --download-db-only` would be smart enough to detect if there was any change between "yesterday's DB" and "today's DB". In other words, it would be possible for the DB not to change, hence not requiring a download.
So the logic is:
1) do not download the DB more than once a day (it can potentially be updated every 6 hours), which is determined based on whether a cache with the exact key already exists
2) if today's DB has not been downloaded yet, call `trivy image --download-db-only` but first get yesterday's DB in case it has not changed since

This is also how the caching function works in `trivy-action`: https://github.com/aquasecurity/trivy-action/blob/18f2510ee396bbf400402947b394f2dd8c87dbb0/action.yaml#L136-L142. In this PR, I didn't want to rely on that "native" caching, as I wanted to support retries for downloading the DB.

Reply:
Got it, thanks for the explaination.